datafusion/expr/src/logical_plan/extension.rs

```diff
@@ -212,8 +212,8 @@ impl Eq for dyn UserDefinedLogicalNode {}
 /// This trait facilitates implementation of the [`UserDefinedLogicalNode`].
 ///
 /// See the example in
-/// [user_defined_plan.rs](../../tests/user_defined_plan.rs) for an
-/// example of how to use this extension API.
+/// [user_defined_plan.rs](<https://github.com/apache/datafusion/blob/main/datafusion/core/tests/user_defined/user_defined_plan.rs>)
```

Comment:
Thanks @dttung2905 !

This is an interesting syntax -- I would expect the `<` and `>` not to be required:

```suggestion
/// [user_defined_plan.rs](https://github.com/apache/datafusion/blob/main/datafusion/core/tests/user_defined/user_defined_plan.rs)
```

Reply:
Thanks @alamb for the review. That was quick! :rocket: